src/plugins/aragon/index.ts

```diff
@@ -1,5 +1,113 @@
-import { sendTransaction } from '../../utils';
-import abi from './DisputableDelay.json';
+import BN from 'bn.js';
+import { keccak256 } from '@ethersproject/keccak256';
+import { toUtf8Bytes } from '@ethersproject/strings';
+import { sendTransaction, subgraphRequest } from '../../utils';
+import abi from './GovernQueue.json';
+
+const ARAGON_SUBGRAPH_URL = {
+  4: 'https://api.thegraph.com/subgraphs/name/evalir/aragon-govern-rinkeby'
+};
+
+const GQL_QUERY = {
```

Comment:
We need to change this query. Now it should be of the form:
```
{
  # id should be the name of the DAO in the registry
  # this will be part of the space config
  registryEntry(id: "autonomous-olive") {
    executor {
      address
    }
    queue {
      address
      config {
        executionDelay
        scheduleDeposit {
          token
          amount
        }
        challengeDeposit {
          token
          amount
        }
        resolver
        rules
      }
    }
  }
}
```